### ISSUE TITLE
ES6 class statics are not transferred to the new class

### ISSUE THREAD

**Issue by panrafal** [brigand/react-mixin#41]
reactMixin.decorate does not transfer static methods declared with static keyword, at least transpiled by Babel:
```js
  @reactMixin.decorate(RegistryMixin)
  class Layer extends React.Component {
    static foo() {
    }
  }
```
That's because `foo` is a not an enumerable property...

**Comment by panrafal** [comment]
Duh, sorry, it works, I wen trigger happy...

**Comment by brigand** [comment]
Fixed and released as 3.0.2.

**Comment by panrafal** [comment]
I can confirm it works, many thanks!
